<ui version="4.0" >
 <class>wifiscanssid</class>
 <widget class="QDialog" name="wifiscanssid" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Scan for Wireless networks</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="wificonfig.qrc" >
    <normaloff>:/tray_wifi85.png</normaloff>:/tray_wifi85.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="textTop" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupWifiScan" >
     <property name="title" >
      <string>Available wireless networks</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="2" column="0" >
       <spacer name="spacer10" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>131</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" >
       <widget class="QPushButton" name="pushRescan" >
        <property name="text" >
         <string>&amp;Rescan</string>
        </property>
        <property name="shortcut" >
         <string>Alt+R</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QListWidget" name="listWifi" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>418</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="spacer8" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>70</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushSelect" >
       <property name="text" >
        <string>Select</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushCancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="wificonfig.qrc" />
 </resources>
 <connections/>
</ui>
